app/controllers/user_controller.rb

```diff
@@ -0,0 +1,8 @@
+class UserController < ApplicationController
+
+  def destroy
+    user = User.find(params[:id])
+    user.destroy
+
+    redirect_to signup_path
```

Comment:
Why are you redirecting the user to the signup page?
Shouldn't you redirect them to the login page instead?